src/Schema/Factories/DirectiveFactory.php

```diff
@@ -150,6 +149,7 @@ public function setResolved(string $directiveName, string $className): self
     }
 
     /**
+     * @deprecated
```

Comment:
What is this for, and why it got deprecated?

Reply:
I don't know, that is exactly why i deprecated it.